doc/advanced-testing-mini-course/src/Lec04FaultInjection.lhs

```diff
@@ -190,7 +199,6 @@ Model
 >     )
 >   ClientRequest (ReadReq (Index ix)) -> (m, Just (ReadResp (mModel m Vector.! ix)))
```

Comment:
Should this make `mMaybeFault` be `Nothing` now?

Reply:
I think so, fixed in next PR.